12/umbraco-cms/content-delivery-api.md

```diff
@@ -0,0 +1,413 @@
+---
+description: Get started with the Content Delivery API.
+---
+
+# Content Delivery API
+
+## Overview
+
+With its several extension points, this new API delivers headless capabilities built directly into Umbraco. It allows you to retrieve your content items in a JSON format and lets you preset them in different channels, using your preferred technology stack. This feature preserves the friendly editing experience of Umbraco, while also ensuring a performant delivery of content, even in headless scenarios.
```

Comment:
Maybe not start the entire article with the mention of extension points; we'd like people to understand that they can use the API out of the box.

```suggestion
The Content Delivery API delivers headless capabilities built directly into Umbraco. It allows you to retrieve your content items in a JSON format and lets you preset them in different channels, using your preferred technology stack. This feature preserves the friendly editing experience of Umbraco, while also ensuring a performant delivery of content in a headless fashion. And with its several extension points, you can tailor this API to fit a broad range of requirements.
```